ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -64,20 +103,25 @@ def uninstall_cluster_logging():
     clusterlogging_obj = ocp.OCP(
         kind=constants.CLUSTER_LOGGING, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
     )
-    assert clusterlogging_obj.delete(resource_name="instance")
+    if clusterlogging_obj:
```

Comment:
You are checking for the python obj existence and not the ocp ressource, so this may always be true,

Reply:
Done